doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -32,14 +32,17 @@ The library is enabled and configured entirely using the Kconfig system.
 Configure the following Kconfig options to enable this library:
 
 * :kconfig:option:`CONFIG_MODEM_SLM` - Enables the Modem SLM library.
-* :kconfig:option:`CONFIG_MODEM_SLM_DMA_MAXLEN` - Configures UART RX EasyDMA buffer size, which is configured to 1024 bytes by default.
+* :kconfig:option:`CONFIG_MODEM_SLM_AT_CMD_RESP_MAX_SIZE` - Configures the size of the AT command response buffer. The default size is 2100 bytes, which is aligned with SLM.
```

Comment:
```suggestion
* :kconfig:option:`CONFIG_MODEM_SLM_AT_CMD_RESP_MAX_SIZE` - Configures the size of the AT command response buffer.
  The default size is 2100 bytes, which is aligned with SLM.
```

Reply:
This Kconfig option is SLM-specific. What does adding `which is aligned with SLM` bring to the table? Is there a particular reason for adding this sentence? @MarkusLassila

Reply:
SLM and Modem SLM are two different things. SLM is the "real" SLM app running on one device and Modem SLM (which this PR modifies) is running on the host device controlling the SLM device.
So this Kconfig is Modem SLM specific, not SLM specific.